BEACON-V2/genomicVariations/defaultSchema.json

```diff
@@ -6,37 +6,21 @@
   "type": "object",
   "required": [
     "variantInternalId",
-    "variantType",
-    "alternateBases",
-    "position"
+    "variation"
   ],
   "properties": {
+    "variation": {
+      "oneOf": [
+        {"$ref": "https://raw.githubusercontent.com/ga4gh/vrs/1.2.0/schema/vrs.json#/definitions/MolecularVariation" },
```

Comment:
I suggest using 1.2 instead of 1.2.0; this will enable use of any patch-level fixes

Reply:
Thanks - modified with https://github.com/ga4gh-beacon/beacon-v2-Models/pull/72/commits/8a0fa5ed3efa320584183bdfb2a5ab39e029a646